Framework/CurveFitting/src/Functions/Bk2BkExpConvPV.cpp

```diff
@@ -75,25 +72,44 @@ double Bk2BkExpConvPV::height() const {
   return height[0];
 }
 
-/** Get peak's FWHM
+/**
+ * Get approximate peak width.
  */
 double Bk2BkExpConvPV::fwhm() const {
-  double sigma2 = this->getParameter("Sigma2");
-  double gamma = this->getParameter("Gamma");
+  // get sigma of Gauss with same FWHM as voigt (H)
   double H, eta;
-  calHandEta(sigma2, gamma, H, eta);
-
-  return mFWHM;
+  calHandEta(getParameter("Sigma2"), getParameter("Gamma"), H, eta);
+  const auto s = H / (2 * sqrt(2 * M_LN2));
```

Comment:
Could we document in comments what were doing here, e.g. a paper / a worked example / or a commonly used operation (e.g. take the mean of x).

This tends to be "obvious" to a reader now, but later on people have to sit and mentally perform the maths each time they work on the code to understand it.

Reply:
Good point - it is actually partially explained on the comment a couple lines above, but I have commented on that line (L82) explicitly what I'm doing (converting FWHM to sigma of a Gaussian).